app/models/framework/sheet.rb

```diff
@@ -0,0 +1,27 @@
+class Framework
+  ##
+  # Define a Sheet for Orders/Invoices on a field-by-field basis
```

Comment:
just wondering if this (and the one for the `Definition` class) could be reworded slightly to be as helpful as possible to someone coming new to the codebase. perhaps just mentioning that they're base/abstract classes would be enough.